aspnetcore/security/docker-https.md

```diff
@@ -68,6 +70,9 @@ When using [PowerShell](/powershell/scripting/overview), replace `%USERPROFILE%`
 
 The password must match the password used for the certificate.
 
+
+Note: The certificate in this case must be a *.pfx* file.  Utilizing a *.crt* or *.key* file with or without the password isn't supported with the sample container.  For example, when specifying a *.crt* file, the container may return error messages such as 'The server mode SSL must use a certificate with the associated private key.'. When using [WSL](/windows/wsl/about), validate the mount path to ensure that the certificate loads correctly.
```

Comment:
```suggestion
Note: The certificate in this case must be a `.pfx` file.  Utilizing a `.crt` or `.key` file with or without the password isn't supported with the sample container.  For example, when specifying a `.crt` file, the container may return error messages such as 'The server mode SSL must use a certificate with the associated private key.'. When using [WSL](/windows/wsl/about), validate the mount path to ensure that the certificate loads correctly.
```

Files should be italic, `*`, but sometimes code fenced looks better. In this case, it's like code.

Reply:
Great, thank you for the suggestion, and updated!